homeassistant/components/sensor/const.py

```diff
@@ -195,6 +195,12 @@ class SensorDeviceClass(StrEnum):
     Unit of measurement: `%`
     """
 
+    IAQ = "iaq"
```

Comment:
Adding an entity device class first requires an approved architecture discussion.

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model